requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit==0.13.3
```

Comment:
Also, we usually don't pin specific versions for our dev dependencies, but that's totally up to you / the project's stance on that.